shipitscript/task.py

```diff
@@ -1,18 +1,35 @@
 import logging
 
-from scriptworker.exceptions import TaskVerificationError
+from scriptworker import client
+from scriptworker.exceptions import TaskVerificationError, ScriptWorkerTaskException
 from scriptworker.utils import get_single_item_from_sequence
 
 
 log = logging.getLogger(__name__)
 
 
-# TODO: Make this prefix a param of the instance config, when Thunderbird migrates this task
-_VALID_SCOPES_PREFIX = 'project:releng:ship-it:'
+# action_map {{{1
+schema_map = {
+    'mark-as-shipped': 'mark_as_shipped_schema_file',
+    'mark-as-started': 'mark_as_started_schema_file',
+}
+
+
+def _get_scope(context, sufix):
+    task = context.task
+    script_config = context.config
```

Comment:
Nit: I think we just want a shorthand for `script_config["taskcluster_scope_prefix"] + suffix` here.